### DISCUSSION TITLE
Is there a way to sync the leveling progress??

### DISCUSSION THREAD

**Discussion by KX5-jeron** [Lailloken/Exile-UI#376] [General]
I started using you overlay when i was already level 72 and just getting into act 10. I can't seem to get it to move past the find and kill Hillock part at the bottom of the screen. I tried manually clicking off all of the tasks on exile leveling site and then exporting that, but still nothing.. Do you have to use the leveling guide from the very start for it to work properly??

**Comment by Lailloken** [comment]
You can long-click the ` > ` button at the bottom of the overlay and it will try to "fast-forward" to the next mention of the area you are currently in.

**Reply by KX5-jeron** [reply]
ah ok that will work, i'll try in when I get home later.. thanks for the quick reply.
